platform/darwin/include/MGLStyle.h

```diff
@@ -4,50 +4,115 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
-/** A collection of convenience methods for creating style URLs of default styles provided by Mapbox. These instances of NSURL are cached. */
+/**
+ A version number identifying the latest released version of the suite of default styles provided by Mapbox. This version number may be passed into one of the “StyleURLWithVersion” class methods of MGLStyle.
+ 
+ The value of this constant is current as of the date on which this SDK was published. Consult the <a href="https://www.mapbox.com/api-documentation/#styles">Mapbox Styles API documentation</a> for the most up-to-date style versioning information.
+ 
+ @warning The value of this constant may change in a future release of the SDK. If you use any feature that depends on a specific implementation detail in a default style, you may use the current value of this constant or the underlying style URL, but do not use the constant itself. Such implementation details may change significantly from version to version.
```

Comment:
How does 1e3b1f2a8fce7d22993deb90e584310dd39b3116 look to you? The constant is now `MGLStyleDefaultVersion`, with documentation that positions it as “here’s what MGLMapView chooses by default” rather than “here’s what you should pass into MGLStyle”. There are no public references to this constant; instead, documentation for each method spells out the current version “as of publication”, and a unit test verifies that it’s in sync with the constant.